### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by HossainAraf**
key Features:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
-  Popups for both mobile and desktop screen sizes.

Method implemented:
- I need to use a JavaScript array to store all of the information for all projects. [Array link](https://github.com/HossainAraf/Portfolio-Mobile-First/blob/2abd5a830b5f896c14974b9da7f41b7575dbfa9a/app.js#L45)
- For each project I need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- I need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- I need to update css and add media queries for different screen size.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -552,6 +603,261 @@ i {
...
+  left: 0;
+  width: 100%;
+  background-color: #c1c7d0;
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  opacity: 0.97;
+  overflow: scroll;
+}
```
Comment (Alusp) [inline style.css:630]: - [ ] Good job so far :clap: However,  On your  `popup` card id for both mobile and desktop,  Please ensure that the font also matches the given design requirement that says `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using
[this design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-418&mode=design&t=Doc6SdyUeETs3nNw-0)
- [ ] Please ensure that you remove the `Lorem Epsum` texts and replace them with your customize texts.


below is an illustration of the difference between yours  and the given design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Desktop


</td>
      <td>

Figma Desktop


</td>
    </tr>
  </table> 


<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Mobile


</td>
      <td>

Figma Mobile Design


</td>
    </tr>
  </table> 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Canopy mobile


</td>
      <td>

Figma mobile Canopy


</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @HossainAraf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
almost

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 
- [ ] Professional Readme used :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HossainAraf** [comment]
Please note that,  some given figma template font properties are not same as figma design. The font doesn't look like the same as the design. I faced the same issue in my previous project. In this whole project only one font family was used. So, there is no chance to conflict with other font families, so far I understand. Thanks.

**Review by usorfaitheloho — APPROVED** [review]
Hi @HossainAraf  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**